tools/perfscale-audit/api/api.go

```diff
@@ -0,0 +1,124 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"time"
+)
+
+type InputConfig struct {
+
+	// StartTime when set, represents the beginning of the metric time range
+	// This defaults to EndTime - Duration when duration is set.
+	StartTime *time.Time `json:"startTime,omitempty"`
+	// EndTime when set, represents end of the metric time range
+	// This defaults to the current time
+	EndTime *time.Time `json:"endTime,omitempty"`
+	// Duration represents how long to go back from EndTime when creating the metric time range
+	// This is mutually exclusive with the StartTime value. Only one of these
+	// two values can be set.
+	Duration *time.Duration `json:"duration,omitempty"`
+
+	PrometheusURL         string `json:"prometheusURL"`
+	PrometheusUserName    string `json:"prometheusUserName"`
+	PrometheusPassword    string `json:"prometheusPassword"`
+	PrometheusBearerToken string `json:"prometheusBearerToken"`
+	PrometheusVerifyTLS   bool   `json:"prometheusVerifyTLS"`
+}
+
+type ResultPhaseTransitionPercentiles struct {
+	P99CreationToRunning float64 `json:"p99CreationToRunning"`
+	P95CreationToRunning float64 `json:"p95CreationToRunning"`
+	P50CreationToRunning float64 `json:"950CreationToRunning"`
```

Comment:
```suggestion
	P50CreationToRunning float64 `json:"p50CreationToRunning"`
```

Reply:
i replaced these with the map suggestion